tests/testsuite/directory.rs

```diff
@@ -762,7 +762,7 @@ fn version_missing() {
 error: failed to compile [..]
 
 Caused by:
-  failed to select a version for the requirement `foo = \"^2\"`
+  failed to select a version for the requirement `foo = \"^2\"` 
```

Comment:
Could this avoid having trailing whitespace in the error messages? (most editors I know of remove trailing whitespace which will otherwise make editing these files pretty difficult